### ISSUE TITLE
Multi-session not working with self-joined users

### ISSUE THREAD

**Issue by SystemKeeper** [nextcloud/spreed#8626]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. User A creates room R with link share (without password) using Firefox
2. User B logs in to nextcloud and opens the link of room R using a private window in Firefox
3. User B logs in to nextcloud and opens the link of room R using Chromium
4. User B closes the browser or changes to another room using Chromium

### Expected behaviour
The session in the private firefox window should still be available and usable.

### Actual behaviour
The session in the private firefox window is redirected to "Conversation not found".

## Talk app

**Talk app version:** Tested with 14.0.7 and latest master (@ 7dc878f1899d646c925ee5b073a93430cf3e40d5)

**Custom Signaling server configured:** Can be reproduced either way

## Remarks
Key to that problem is that the user is not added by a moderator to the group conversation, but "self-joins" via a public link. When leaving one conversation, the attendee is removed here: https://github.com/nextcloud/spreed/blob/c4e47dcb3d586c860e00b1f50ffa26b2f20d1778/lib/Service/ParticipantService.php#L775-L780 which then results in the "Conversation not found" redirect.

**Comment by nickvergessen** [comment]
Yeah, basically we need to check if there are any sessions remaining before deleting the attendee.
